misc/quoter-core/build.gradle

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Are there another component as quoter-* other than quoter-core? If not, shall we rename this to `api-quoter`

Reply:
Initially there were `quoter-cli` and `quoter-server` since this project was initially supposed to be a CLI tool.